### ISSUE TITLE
change age to birth year

### ISSUE THREAD

**Issue by suhlrich** [stanfordnmbl/opencap-viewer#106]
change age to birth year...age will be inaccurate over time. both api and viewer.

**Comment by antoinefalisse** [comment]
@suhlrich and @AlbertoCasasOrtiz what do you think is best here?

1. Keep the same field in the database, but give it a different meaning (year of birth instead of age)
2. Add a new field in the database and no longer use `age`

In both cases, we could assume all subjects were created in 2023 and accordingly modify (option 1.) or populate (option 2.) the database.

**Comment by AlbertoCasasOrtiz** [comment]
I think creating a new field and not using age anymore would be best.

We could then use age and date of creation of the first session associated to a specific subject to calculate the year o of birth of existing subjects and populate it.

**Comment by antoinefalisse** [comment]
@anastasiiadidan and @olehkorkh-planeks, when we added the subject feature, we added the field "Age". What we want instead is the year of birth, so that we can for instance compare data when the subject was 20, 23, and 25 years old. Let's just modify what we did:

**Frontend**: 
- replace "Age" by "Birth Year"

**API**: 
- add "Birth Year" to the database
- for existing subjects with an "Age" entry, automatically populate the "Birth Year" entry. We released the feature in 2023, so if a subject has "Age" 32, then "Birth Year" is 1991 (2023 - 32).

Let us know for questions
